src/main/java/cz/cvut/kbss/termit/service/business/AssetService.java

```diff
@@ -65,6 +66,20 @@ public AssetService(ResourceRepositoryService resourceRepositoryService,
         return result.subList(0, Math.min(result.size(), limit));
     }
 
+    /**
+     * Finds the specified number of most recently commented assets.
+     *
+     * @param limit Maximum number of assets to retrieve
+     * @return List of recently commented assets
+     */
+    public List<RecentlyCommentedAsset> findLastCommented(int limit) {
+        if (limit < 0) {
+            throw new IllegalArgumentException("Maximum for recently commented assets must not be less than 0.");
+        }
```

Comment:
This fragment appears three times in this class. Should be extracted into a method.